test/test-utils/index.test.tsx

```diff
@@ -4,9 +4,20 @@ import React, { useState } from 'react';
 import { FormValidation, Morfi } from 'morfi';
 import { MorfiTestUtils } from '../../test-utils/index'; // cannot be shortened because of nested package.json
 import { act, render, cleanup } from '@testing-library/react';
-
-type FormValues = { age: number; name: string; displayName?: string };
-const initialValues: FormValues = { age: 0, name: '' };
+Morfi.setup({
+    comparator(val, val2){
+        if(val instanceof Date && val2 instanceof Date) return val.toString() === val2.toString()
+        return val === val2;
+    }
+})
+
+function getCurrentDateWithoutTime(){
```

Comment:
prefer arrow functions and use a stable data source. Your code could possibly fail if invoked in very unlucky moment in time. Use rather static values like `new Date(2023,1,1)` etc.